projects/plugins/jetpack/changelog/add-instant-search-alt-text-where-available#2

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
Do you think keeping `projects/plugins/jetpack/changelog/add-instant-search-alt-text-where-available` is sufficient enough?